modules/core/nuxeo-core-api/src/main/java/org/nuxeo/ecm/core/api/validation/ConstraintViolation.java

```diff
@@ -96,6 +96,17 @@ public Schema getSchema() {
         return Collections.unmodifiableList(path);
     }
 
+    public String getPathAsString() {
+        StringBuilder sb = new StringBuilder();
+        for (int i = 0; i < path.size(); i++) {
+            sb.append(path.get(i).toString());
+            if (i != path.size() - 1) {
+                sb.append("/");
+            }
+        }
+        return sb.toString();
+    }
+
```

Comment:
Couldn't it be simplified to something like

```suggestion
    public String getPathAsString() {
        path.stream().map(PathNode::toString).collect(Collectors.joining("/"));
    }
```
?